.github/workflows/stale.yml

```diff
@@ -29,4 +29,4 @@ jobs:
         stale-pr-label: 'stale'
         close-issue-message: 'This issue has been closed automatically due to a lack of activity.'
         close-pr-message: 'This pull request has been closed automatically due to a lack of activity.'
-        days-before-stale: 30
+        days-before-pr-stale: 30
```

Comment:
What about `days-before-issue-stale`? Is the default 60 days?

Reply:
Yes, default is 60 days. However, I felt 60 days was a bit much for pull requests.

Reply:
And I intended to use `dats-before-pr-stale` in the first place but copied the wrong setting...